android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -68,6 +68,16 @@
 import io.sentry.Sentry;
 import io.sentry.SentryLevel;
 
+class JSQueueEntry {
+  public String call;
+  public boolean pause;
+
+  JSQueueEntry(String call, boolean pauseAfterCall) {
+    this.call = call;
+    this.pause = pauseAfterCall;
+  }
+}
```

Comment:
Given we ended up never using `pause`, I suggest we remove this again and go back to a `ArrayList<String>` :grin:  ... which will remove most of the changes from this PR also!

Reply:
**If** we decided to keep the plumbing for pause, maybe an override would be minimize changes:

```
public void loadJavascript(String func) {
  // Default pauseAfterCall to false
  loadJavascript(func, false);
```